pkgs/tools/security/networkminer/default.nix

```diff
@@ -0,0 +1,73 @@
+{ lib
+, buildDotnetModule
+, fetchurl
+, unzip
+, dos2unix
+, makeWrapper
+, msbuild
+, mono
+}:
+buildDotnetModule rec {
+  pname = "networkminer";
+  version = "2.8";
+
+  src = fetchurl {
+    # Upstream does not provide versioned releases, a mirror has been uploaded
+    # to archive.org
+    url = "https://archive.org/download/networkminer-${builtins.replaceStrings ["."] ["-"] version}/NetworkMiner_${builtins.replaceStrings ["."] ["-"] version}_source.zip";
+    sha256 = "1n2312acq5rq0jizlcfk0crslx3wgcsd836p47nk3pnapzw0cqvv";
+  };
+
+  nativeBuildInputs = [ unzip dos2unix msbuild ];
+
+  patches = [
+    # Store application data in XDG_DATA_DIRS instead of trying to write to nix store
```

Comment:
Can we try to upstream this?

Reply:
The upstream project does not appear to accept contributions...